.storybook/main.js

```diff
@@ -16,14 +16,17 @@ module.exports = {
     './i18n-party-addon/register.js',
     'storybook-dark-mode',
   ],
+  staticDirs: ['../app', './images'],
```

Comment:
Replaces static directory flags in `package.json`
https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#deprecated---static-dir-cli-flag